app/apps/server/communication/rest.js

```diff
@@ -57,15 +57,15 @@ export class AppsRestApi {
 		const manager = this._manager;
 		const multipartFormDataHandler = this._handleMultipartFormData;
 
-		const handleError = (message, e) => {
+		const handleWarn = (message, e) => {
 			// when there is no `response` field in the error, it means the request
 			// couldn't even make it to the server
 			if (!e.hasOwnProperty('response')) {
-				orchestrator.getRocketChatLogger().error(message, e.message);
+				orchestrator.getRocketChatLogger().warn(message, e.message);
 				return API.v1.internalError('Could not reach the Marketplace');
 			}
 
-			orchestrator.getRocketChatLogger().error(message, e.response.data);
+			orchestrator.getRocketChatLogger().warn(message, e.response.data);
```

Comment:
Ok, it makes sense. I thought of changing this _error_ into a _warn_ because it covers the case where an user doesn't have access to the internet but the server does. But thinking right now it's more reasonable to throw an _error_ then a _warn_, my bad!